pkgs/by-name/pa/passbolt/package.nix

```diff
@@ -0,0 +1,49 @@
+{ lib,
+fetchFromGitHub,
+php,
+dataDir ? "/var/lib/passbolt",
+runtimeDir ? "/run/passbolt"
+}:
+
+php.buildComposerProject (finalAttrs: {
+  pname = "passbolt_api";
+  version = "4.5.2";
```

Comment:
there is a new version now 
https://github.com/passbolt/passbolt_api/releases/tag/v4.9.1
maybe there is a chance a bug that was blocking this build go fixed?

Reply:
Will try it out, sorry it seems I missed your comment

Reply:
It does work with 4.9.1, despite the warning I had before, will bump the version, however I'm very busy at the moment, I won't be able to finish this PR, but I may work on it slowly but surely towards something at some point, I guess